ownCloud/Client/BreadCrumbTableViewController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  BreadCrumbTableViewController.swift
+//  ownCloud
+//
+//  Created by Matthias Hühne on 09.04.19.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2019, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import UIKit
+
+class BreadCrumbTableViewController: StaticTableViewController {
+
+	// MARK: - Constants
+	private let contentWidth = 500
+	private let rowHeight = 44
+	private let imageWidth = 30
+	private let imageHeight = 30
+
+	// MARK: - Instance Variables
+	var parentNavigationController : UINavigationController?
+	var queryPath : NSString = ""
+	var bookmarkShortName : String?
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		self.tableView.isScrollEnabled = false
+		guard let stackViewControllers = parentNavigationController?.viewControllers else { return }
+		var pathComp = queryPath.pathComponents
+
+		if queryPath.hasSuffix("/") {
+			pathComp.removeLast()
+		}
+		if pathComp.count > 1 {
+			pathComp.removeLast()
+		}
+
+		var rows : [StaticTableViewRow] = []
+		let counter = pathComp.count
```

Comment:
@hosy I would prefer more descriptive names than just `counter` or `iterator` (so `rowHeight` as an example is much clearer than just `height`), also adding 'er' suffix does make them sounding as object instance names capable of performing complex actions.